contrib/cirrus/runner.sh

```diff
@@ -51,6 +51,10 @@ function _run_validate() {
     bin/podman --version
     bin/podman-remote --version
     make validate  # Some items require a build
+
+    # TODO: Remove the next line once FindBin included in VM image
+    dnf install -y perl-FindBin
+    make xref_helpmsgs_manpages  # always require a complete build
```

Comment:
Please remove these, I've solved the problem a different way in #8299. Once these lines are gone, I can rebase on your PR and try out CI-colon-docs. Thanks.